include/swift/SwiftRemoteMirror/Platform.h

```diff
@@ -18,7 +18,7 @@ extern "C" {
 #endif
 
 #if defined(swiftRemoteMirror_EXPORTS)
-# if defined(__ELF__)
+# if defined(__ELF__) || defined(__wasi__)
```

Comment:
I think that this should be `__WASM__` as the object file format is WASM, not WASI.